app/home/page.tsx

```diff
@@ -1,15 +1,41 @@
-import React from 'react'
+import React from "react";
+import HomeCard from "@/components/home_card/HomeCard";
+import background from "@/images/background.png";
+import box1 from "@/images/boxes/box1.jpg";
+import box2 from "@/images/boxes/box2.jpg";
+import box3 from "@/images/boxes/box3.jpg";
+import box4 from "@/images/boxes/box4.jpg";
 
 function Home() {
   return (
-    <div className="h-full">
-      <div className="flex h-full">
-        <div className="w-1/2 bg-slate-700 float-left text-center">
-            Home...
+    <div className="flex justify-center">
```

Comment:
You could add "h-full" so that larger screens/page zoom dont have white space underneath

Reply:
@Lunchii What does that fix? I don't see any difference.